packages/fcl/src/exec/mutate.md

```diff
@@ -217,3 +219,28 @@ const txId = await fcl.mutate({
 ```
 
 More specific options overload less specific options. So `payer` will be used over `authz` which will be used over the default of current user.
+
+## Interaction Templates
+
+If you have an Interaction Template, you can use it with `mutate`:
+
```

Comment:
Add headings for the two options for including a template

## Pass template string

## Pass a URL